src/content/docs/en/guides/integrations-guide/db.mdx

```diff
@@ -92,6 +92,31 @@ export default defineDb({
 })
 ```
 
+## Configuration
+
+### `mode`
+
+Astro DB supports different driver modes for production deployments, by default it uses a node-based libSQL driver. When deploying to a serverless environment's that don't support all features of Node.js, such as Cloudflare Workers, you can switch to the web libSQL driver allowing for Astro DB to continue functioning in these environments.
+
+To configure the driver mode, set the `mode` property in your `astro.config.mjs` file:
```

Comment:
```suggestion
To use the web libSQL driver mode for non-Node.js environments, set the `mode` property in your adapter's configuration:
```
In the case where there's only one non-default option, we can be a bit more explicit about saying "configure for this one case". This also lets us avoid repeating words when trying to speak in generalities! ("Configure mode to configure the driver mode in your configuration file)